.env.sample

```diff
@@ -49,4 +49,7 @@ ORCHESTRATION_STRATEGY=openai_functions
 #Speech-to-text feature
 AZURE_SPEECH_SERVICE_KEY=
 AZURE_SPEECH_SERVICE_REGION=
-AZURE_AUTH_TYPE=rbac
\ No newline at end of file
+# Auth type related param. When your AZURE_AUTH_TYPE = rbac, please make sure setting USE_KEY_VAULT= false; When your USE_KEY_VAULT= true, please setting your key vault endpoint.
```

Comment:
```suggestion
# Auth type environment variables.
# When AZURE_AUTH_TYPE=rbac, please make sure variable USE_KEY_VAULT=false
# When USE_KEY_VAULT = true, please make sure to set AZURE_KEY_VAULT_ENDPOINT
```

Rewording for clarity.


FYI, for azure-search-openai-demo, we sometimes have situations where we use RBAC for the majority but keyvault for a few SDKs, due to lack of RBAC support for that SDK or SKU. So you may imagine a world where that happens for this SDK, and the key vault endpoint isn't mutually exclusive. Fine for now though if you managed to make a clean separation!